indexer/lib/fwss-handlers.js

```diff
@@ -50,7 +50,7 @@ export async function handleFWSSDataSetCreated(
         with_ipfs_indexing
       )
       VALUES (?, ?, ?, ?, ?)
-      ON CONFLICT DO NOTHING
+      ON CONFLICT (id) DO NOTHING
```

Comment:
I think we can revert this back to the original version. Thoughts?


```suggestion
      ON CONFLICT DO NOTHING
```